### PR TITLE
Updated pipe visual demos to use the browser's current locale

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Just adding a nicety to the visual demos here. This will allow you to change your browser's language and test the pipes in a more realistic scenario.

It might be nice to do this to the datepickers at some point.

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-visual-test-pipe-update](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-visual-test-pipe-update)

File: src/app/visual/date-pipe/date-pipe-visual.component.ts [hunk 1]
```diff
@@ -46,15 +55,30 @@ export class DatePipeVisualComponent implements OnInit {
...
+  private ngUnsubscribe = new Subject<void>();
+
   constructor(
-    private datePipe: SkyDatePipe
-  ) { }
+    private datePipe: SkyDatePipe,
+    private localeProvider: SkyAppLocaleProvider
+  ) {
+    // Update locale to the browser's current locale.
+    this.localeProvider.getLocaleInfo()
```
Comment (Blackbaud-SteveBrush) [inline src/app/visual/date-pipe/date-pipe-visual.component.ts]: Isn't this done automatically by the pipe (and Builder)?
See: https://github.com/blackbaud/skyux-datetime/blob/master/src/app/public/modules/date-pipe/date.pipe.ts#L44-L49

I thought the public `locale` variable in the visual demo was being used as an override to the browser's current culture?

Comment (Blackbaud-AlexKingman) [reply src/app/visual/date-pipe/date-pipe-visual.component.ts]: > I thought the public locale variable in the visual demo was being used as an override to the browser's current culture?

Correct. The only thing the demos weren't doing was showing you (in the template) what locale you were currently using on load. For example, the dropdown menu should now show your current locale and the displayed formatting will respect the browser's current locale on load. Before this change, it was always initializing with `en-US`.

Comment (Blackbaud-SteveBrush) [reply src/app/visual/date-pipe/date-pipe-visual.component.ts]: Even if the `locale` was left `undefined`?

Comment (Blackbaud-AlexKingman) [reply src/app/visual/date-pipe/date-pipe-visual.component.ts]: If the `locale` is `undefined` the pipe works correctly (shows the browser's current locale`, but the visual test doesn't show which local is selected in the dropdown. That's really the only reason I did this - it was just a nicety.

Maybe overkill?

**Pull request closed without merging**